src/file-events/cpp/generic_fsnotifier.cpp

```diff
@@ -282,9 +211,8 @@ JNIEXPORT void JNICALL Java_net_rubygrapefruit_platform_internal_jni_AbstractFil
     logging->invalidateLogLevelCache();
 }
 
-JniConstants::JniConstants(JavaVM* jvm)
+NativeConstants::NativeConstants(JavaVM* jvm)
```

Comment:
How are those native constants? Aren't these constants of things on the Java side?

Reply:
`Native` (with capital `N`) is used to refer to the whole of `native-platform`.

Reply:
I would prefer having only `JniConstants`.

If you want to keep both classes, I think `NativePlatformJniConstants` would be a better name.

Reply:
`NativePlatformJniConstants` it is now!